packages/components/src/define-config.tsx

```diff
@@ -20,6 +22,7 @@ export function defineConfig({
   const siteName = product
     ? `${['ANGULAR', 'KITQL'].includes(originalSiteName) ? '' : 'GraphQL '}${product.name}`
     : originalSiteName;
+  const siteUrl = process.env.SITE_URL;
```

Comment:
What if this env var is non existent? Then generated urls will be `undefined/SOMETHING`?

Reply:
Should be not exist according this line https://github.com/the-guild-org/docs/blob/c96694def3a4b3c838863d409e539322cb7e1ccb/packages/components/src/define-config.tsx#L84